WorkflowAdminBundle/Resources/public/ecmascript/OpenOrchestra/Application/Router/WorkflowProfile/WorkflowProfileRouter.js

```diff
@@ -61,7 +61,7 @@ class WorkflowProfileRouter extends AbstractWorkflowRouter
         let url = Routing.generate('open_orchestra_workflow_admin_workflow_profile_new');
         this._displayLoader(Application.getRegion('content'));
         FormBuilder.createFormFromUrl(url, (form) => {
-            let newWorkflowProfileFormView = new NewWorkflowProfileFormView({
+            let newWorkflowProfileFormView = new WorkflowProfileFormView({
```

Comment:
Could you rename newWorkflowProfileFormView to workflowProfileFormView to keep same logic as in other router?